### PR TITLE
Fixed typos

### PR DESCRIPTION
**Pull request by SaloniThete**
Fixed typos

### CODE REVIEW

**Review by marqbeniamin — CHANGES_REQUESTED** [review]
Hi, thank you for your PR. Please revert Next to Nuxt. Nuxt is for vuejs, and next is for react js.

**Comment by SaloniThete** [comment]
Thanks for letting me know. I have made the requested changes.

File: ROADMAP.md [hunk 1]
```diff
@@ -200,7 +200,7 @@ I will point out those changes inside the migration guide.
 - Touch Events
 - Scrolling
 - Optimized theme colors
-- Vuelidate and Vee Validate integration
```
Comment (marqbeniamin) [inline ROADMAP.md:203]: This vuelidate is a js module for validation: https://vuelidate.js.org/, So you need to revert it

**Review by marqbeniamin — CHANGES_REQUESTED** [review]

File: ROADMAP.md (hunk 1 above)
Comment (SaloniThete) [reply ROADMAP.md]: yes the change is done.

**Review by marqbeniamin — APPROVED** [review]

**Pull request merged**